wiki/Conceptualising Artificial Intelligence as a Digital Healthcare Innovation  - Finn Bracklow.qmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Conceptualising Artificial Intelligence as a Digital Healthcare Innovation: An Introductory Review"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: Finn Bracklow
+categories:
+  - "Digital Healthcare"
+  - "Digital Media Use (AI)"
+  - "Risks of Atonomylose"
+---
+
+Key Quote 
+
+"Artificial intelligence (AI) is widely recognised as a transformative innovation and is already proving capable of outperforming human clinicians in the diagnosis of specific medical conditions, especially in image analysis within dermatology and radiology."
+
+Argument
+
+AI is capable of supporting and even replacing the work of doctors and medical professionals. This is because AI is able to learn from patient records, genomic information, and real-time patient data to identify patterns and characteristics that cannot be detected by humans. For Example AI can help diagnose diseases accurately and promptly.
```

Comment:
Can you focus the argument more on the relationship of using this technology and (individuals') autonomy need experiences?